### ISSUE TITLE
interacted spawner objects existed in the same position when go to a new room by Enable Fast Room Switching from preference. 

### ISSUE THREAD

**Issue by qiangjiadong** [mozilla/hubs#5382]
**Description**

**To Reproduce**
1. Go to 'scene 1 with spawner objects'
2. Click on 'spawner objects and clone objects'
3. Go to 'scene 2 by Enable Fast Room Switching from preference'
4. See error: interacted spawner objects existed in the same position

**Expected behavior**
interacted spawner objects not inherited from the last room

**Screenshots**

**Hardware** 
 - Device: [e.g. Desktop, phone, VR headset, all]
 - OS: [e.g. Windows, iOS, Linux]
 - Browser: [e.g. Firefox and chrome]

**Additional context**
N/A

**Comment by cornelionce** [comment]
I was unable to reproduce this using my instances on a Windows 10 x64 machine. The spawned objects are not displayed in the new room (fast switch enabled).
I've tested using latest Firefox and Chrome browsers.

@qiangjiadong, could you please provide more information about it (i.e. deviced used, browser, OS)?
It is always reproducing on your end? 
In this case it would help if you can give us a link to the rooms you've used or post a screencast with the issue.

**Comment by qiangjiadong** [comment]
I use the latest Chrome on Windows 10 x64, it is always reproducing on my end, 
https://hubs.mozilla.com/eF4iF2k pls use this link with fast switch enabled
Pls see this gameplay video for more info on room link and interactable object https://youtu.be/Kv7v7wRl0I8
Thanks a lot!

**Comment by cornelionce** [comment]
Thanks for the information, it was very useful in helping me to reproduce this behavior using the room you've provided.

It indeed seems to be related to Fast Room Switching, it does not reproduce with the option disabled.

**Comment by qiangjiadong** [comment]
Thks a lot and let me know if you need more info, really appreciate it!

**Comment by Exairnous** [comment]
As far as I know, this is an intentional feature of Fast Room Switching, although I think it was left open to reevaluation.

**Comment by qiangjiadong** [comment]
Thks! I kind of agree. From my case, as I have in total 14 rooms to be connected with fast switch, which will cause too many objects inherited from the last.

**Comment by Exairnous** [comment]
Yeah, when I first noticed it I thought it was a bug too, but it is sort of nice to be able to bring objects with you from room to room.  But it's also true that there isn't any spatial connectivity with you (they just keep their current world coordinates which can make them show up in totally arbitrary places in the new room) and the objects aren't necessarily relevant to the new room.  I wish you could parent objects to your avatar and drag them along with you, and I think those would make a lot of sense to bring from room to room.  All this to say I'm not really sure what the behavior should be :shrug:

**Comment by emclaren** [comment]
Fast room switching still has some bugs and needs further testing, which is why it is off by default. We will address this when we make it a default setting.
